src/rust/protocol/src/memcache/wire/mod.rs

```diff
@@ -11,15 +11,22 @@ pub use response::*;
 use super::*;
 use crate::*;
 
-use metrics::{static_metrics, Counter};
+use metrics::{static_metrics, Counter, Relaxed, Heatmap};
+use rustcommon_time::{Duration, Instant};
 
 static_metrics! {
     static GET: Counter;
+    static GET_CARDINALITY: Relaxed<Heatmap> = Relaxed::new(||
```

Comment:
Open questions:
- Should we call this 'cardinality' or 'batch_size'? 
- Do we want to track GET/GETS together or separately?

Reply:
I think `gets` will mostly be unbatched, which may skew the results slightly? (`gets` is used mostly for update with `cas`, and batch updates are uncommon). I am OK if we *only* do this for `get` now.

Reply:
no opinion on the naming, leaning slightly  toward `cardinality` since it's one word 😄